charts/prometheus-rabbitmq-exporter/templates/prometheusrule.yaml

```diff
@@ -12,36 +12,55 @@ metadata:
   {{- end }}
 spec:
   groups:
-  - name: rabbitmq
-    rules:
-    - alert: RabbitmqDown
-      expr: |
-        absent(rabbitmq_up == 1)
-      for: 1m
-      labels:
-        severity: critical
-    - alert: RabbitmqNotRunning
-      expr: |
-        rabbitmq_running{self="1"} != 1
-      for: 1m
-      labels:
-        severity: critical
-    - alert: RabbitmqMessages
-      expr: |
-        rabbitmq_queue_messages > 250
-      for: 10m
-      labels:
-        severity: warning
-    - alert: RabbitmqMessages
-      expr: |
-        rabbitmq_queue_messages > 500
-      for: 10m
-      labels:
-        severity: critical
-    - alert: RabbitmqPartition
-      expr: |
-        rabbitmq_partitions{self="1"} != 0
-      for: 1m
-      labels:
-        severity: critical
+    - name: {{ template "prometheus-rabbitmq-exporter.fullname" . }}
+      rules:
+        - alert: RMQNodeDown
```

Comment:
Should we stick with the previous naming, like `RabbitmqNodeDown`, which is more explicit ? So that it is closer than what we can see elsewhere, like https://awesome-prometheus-alerts.grep.to/rules.html#rabbitmq ?

Reply:
Done, `Rabbitmq` everywhere